app/cdap/components/ConnectionList/Components/ConnectionTabs/index.tsx

```diff
@@ -87,44 +95,19 @@ export default function ConnectionsTabs({
               root: classes.tabsContainer,
             }}
           >
-            {tabsData.data.map((connectorType, connectorTypeIndex) => (
+            {tabsData?.data?.map((connectorType: IConnectorTabType, connectorTypeIndex: number) => (
```

Comment:
either rename connectorType to eachTabData and connectorTypeIndex to tabIndex
otherwise rename tabsData to something similar to connectorTypes plz

plz maintain consistency in code and practises as well

Reply:
Done